_fr/getting-started/index.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: singlepage-overview
+title: Démarrage
+partof: getting-started
+language: fr
+includeTOC: true
+---
+
+Les instructions ci-dessous couvrent à la fois Scala 2 et Scala 3.
+
+## Essayer Scala sans installation
+
+Pour commencer à expérimenter Scala sans plus attendre, utilisez <a href="https://scastie.scala-lang.org/pEBYc5VMT02wAGaDrfLnyw" target="_blank">“Scastie” dans votre navigateur</a> _Scastie_ est un environnement "bac à sable" en ligne, où vous pouvez tester Scala, afin de comprendre comment fonctionne le langage et avec un accès à tous les compilateurs Scala et les librairies publiées.
+
+> Scastie supporte à la fois Scala 2 et Scala 3, en proposant Scala 3 par défaut.
+> Si vous cherchez à tester un morceau de code avec Scala 2
+> [cliquez ici](https://scastie.scala-lang.org/MHc7C9iiTbGfeSAvg8CKAA).
+
+## Installer Scala sur votre ordinateur
+
+Installer Scala veut dire installer différents outils en ligne de commande, comme le compilateur Scala et les outils de build.
+Nous recommandons l'utilisation de l'outil d'installation "Coursier" qui va automatiquement installer toutes les dépendances, mais vous pouvez aussi installer chaque outil à la main.
+
+### Utilisation de l'installateur Scala (recommandé)
+
+L'installateur Scala est un outil nommé [Coursier](https://get-coursier.io/docs/cli-overview), la commande principale de l'outil est `cs`.
+Il s'assure que la JVM est les outils standards de Scala sont installés sur votre système. 
+Installez-le sur votre système avec les instructions suivantes.
+
+<div class="main-download">
+    <div id="download-step-one">
+        <p>Suivez <a href="https://get-coursier.io/docs/cli-installation.html#native-launcher" target="_blank">les instructions pour installer la commande <code>cs</code></a> puis exécutez :</p>
+        <p><code>$ ./cs setup</code></p>
+    </div>
+</div>
+
+En plus de gérer les JVMs, `cs setup` installe aussi des utilitaires en ligne de commande :
+
+- Un JDK (si vous n'en avez pas déjà un)
+- L'outil de construction de package [sbt](https://www.scala-sbt.org/)
```

Comment:
According to Wikipedia, the french translation of "build tool" is "moteur de production". I don't think this is necessarily clearer, but that avoids franglais. I'm sure @sjrd has a better suggestion?